documentation/outlet-control/index.rst

```diff
@@ -0,0 +1,110 @@
+.. _Outlet-control-doc:
+
+**************
+Outlet control
+**************
+
+.. warning:: Outlet control operations will affect connected devices. These operations have the same effect as the corresponding physical actions. It is the responsibility of the user to assess the situation and act accordingly.
+
+Hyperview Outlet Control has been designed to help you manage and control the power state of compatible Smart Power Distribution Units (rPDUs) while maintaining a detailed record of all actions taken by users.
+
+This feature allows you to:
+
+- Turn on compatible rPDUs - Remotely power up individual outlets, ensuring that power is allocated only when and where it is required.
+
+- Turn off compatible rPDUs - Remotely power down individual outlets, allowing you to save energy and reduce operational costs.
+
+- Power cycle compatible rPDUs - Remotely perform power cycling on individual outlets, enabling you to reset unresponsive equipment and troubleshoot potential issues without the need for onsite intervention.
+
+Work Orders maintain a comprehensive paper trail of all control requests made by users, allowing you to review historical actions and ensure accountability for any changes made.
```

Comment:
Nitpick, wondering if audit trail instead of paper trail better describes it? I don't feel too strongly about it.

Reply:
thanks! 
updated ..